qr-code/src/test/java/com/adyen/checkout/qrcode/internal/ui/DefaultQRCodeDelegateTest.kt

```diff
@@ -385,38 +456,154 @@ internal class DefaultQRCodeDelegateTest(
     }
 
     @Test
-    fun `test download qr image with successful result`() = runTest {
-        delegate.initialize(CoroutineScope(UnconfinedTestDispatcher()))
+    fun `test refreshStatus`() = runTest {
```

Comment:
Is it possible to give these test another name that follows the `when..., then...` format maybe? 😬